src/data.ts

```diff
@@ -1,9 +1,14 @@
 import emojilib from 'emojilib'
 
-import { normalizeCode } from './utils.js'
+import { normalizeCode } from './utils'
```

Comment:
Removing the `.js` from import paths isn't necessary for TypeScript ... but `@swc/jest` is easier to configure without them 😄. And, hey, it's less code!